cmd/osbuild-worker/config.go

```diff
@@ -71,6 +76,7 @@ type workerConfig struct {
 	GenericS3      *genericS3Config            `toml:"generic_s3"`
 	Authentication *authenticationConfig       `toml:"authentication"`
 	Containers     *containersConfig           `toml:"containers"`
+	Pulp           *pulpConfig                 `toml:"pulp"`
```

Comment:
It would be good to extend the extend the unit test for config parser:
https://github.com/osbuild/osbuild-composer/blob/eb8b63f3ca166ecad13e10a21b8499344a824f86/cmd/osbuild-worker/config_test.go#L11

a follow up is OK...

Reply:
I'll do it here.